### PR TITLE
Adding GWF RCM-LRW10 device

### PR DESCRIPTION
**Pull request by maebli**
**Summary**

Adding gwf module rcm-lrw10 with parser.

**Changes**

Vendor added "GWF Messsysteme AG" ( vendor id 179)
Parser and RCM-LRW10 device info added

### CODE REVIEW

**Review by Jaime-Trinidad — APPROVED** [review]

File: vendor/gwf/rcm-lrw10-868.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+vendorProfileID: 01
+supportsClassB: false
+supportsClassC: false
+macVersion: 1.0.1
```
Comment (Jaime-Trinidad) [inline vendor/gwf/rcm-lrw10-868.yaml:4]: Hi @maebli you should add `regionalParametersVersion`

Comment (maebli) [reply vendor/gwf/rcm-lrw10-868.yaml:4]: @Jaime-Trinidad thanks for the review!

I thought that for macVersion 1.0.1 the Regional Parameters are not a separate document. 
I saw the following in another vendor folder:

```
# LoRaWAN Regional Parameters version. Values depend on the LoRaWAN version:
#   1.0:   TS001-1.0
#   1.0.1: TS001-1.0.1
#   1.0.2: RP001-1.0.2 or RP001-1.0.2-RevB
#   1.0.3: RP001-1.0.3-RevA
#   1.0.4: RP002-1.0.0 or RP002-1.0.1
#   1.1:   RP001-1.1-RevA or RP001-1.1-RevB
regionalParametersVersion: RP001-1.0.2
```

and I tried to build using
regionalParametersVersion: TS001-1.0

but the validator script failed, so I removed it completely. I will try again

Comment (maebli) [reply vendor/gwf/rcm-lrw10-868.yaml:4]: The following screenshot shows what got me confused "release device repository" action is passed but "validate" action is failed. Probably I misunderstand the actions, but I find it somewhat confusing that they don't both pass. When I remove the regionalParameter then I get a pass on "validate" but not on "release device repository" 
grafik

Comment (Jaime-Trinidad) [reply vendor/gwf/rcm-lrw10-868.yaml:4]: > The following screenshot shows what got me confused "release device repository" action has passed but "validate" action has failed. Probably I misunderstand the actions, but I find it somewhat confusing that they don't both pass. When I remove the regionalParameter then I get a pass on "validate" but not on "release device repository" grafik

let me check

**Comment by Jaime-Trinidad** [comment]
@maebli The image is throwing a corrupted file error, can you check it please

**Review by Jaime-Trinidad — APPROVED** [review]

**Pull request merged**